src/secret/types.rs

```diff
@@ -88,11 +93,61 @@ pub struct LedgerStatus {
     pub(crate) app: Option<LedgerApp>,
 }
 
+///
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct OutputMetadata {
```

Comment:
Should have `#[serde(rename_all = "camelCase")]` or each field renamed like that

Reply:
Yeah I asked on Slack if we could nest all these into a `metadata` field because otherwise we have to redo this all over again :/

Reply:
Will do in the meantime